android/app/src/main/res/values/strings.xml

```diff
@@ -190,6 +190,7 @@
     <string name="dismissed_from_the_app" formatted="false">Dismissed from the app on %s</string>
     <string name="dismissed_from_notification" formatted="false">Dismissed via notification %s</string>
     <string name="event_moved_new_time" formatted="false">Moved, new time: %s</string>
+    <string name="event_rescheduled_new_time" formatted="false">Confirmed Rescheduled on %s</string>
```

Comment:
We should add the base time to these dogs and potentially even the database. We can probably get it out of the meta thing so it's a confirmed rescheduled after whatever base time

Reply:
changed my mind. we care most about when it was rescheduled to (not as much about when it was rescheduled after aka the base time.)

when it was reschedule to allows us to reason about where it is on the calendar now. the base time would be kinda useful but more just in an informational sense and plus we don't already have it

Reply:
this is not trivial and I think beyond the scope of this pr checkout docs/dev_todo/reschedule_confirmation_handling.md